snare/tests/test_get_dorks.py

```diff
@@ -45,6 +46,26 @@ def test_return_dorks(self):
         self.loop.run_until_complete(test())
         self.assertEquals(self.data, self.dorks['response']['dorks'])
 
+    def test_get_dorks_error(self):
```

Comment:
please mention logging in the test name if your aim is to check logging functionality :)

Reply:
Done.